docs/source/reference/command_line.rst

```diff
@@ -179,7 +179,11 @@ Below is a list with all available subcommands.
       Inspect, create and manage data nodes.
 
     Options:
-      --help  Show this message and exit.
+      -v, --verbosity LEVEL  Control the verbosity of messages emitted by the command by
```

Comment:
Why is this only added to the `verdi data` command?

Reply:
That is a good question... ;-)